### PR TITLE
Taxes listed on an invoice

### PR DESCRIPTION
**Pull request by GSadee**
screencapture-localhost-8080-admin-invoices-fefef005-aaff-4999-956c-4df130431a21-2018-07-10-18_00_19

### CODE REVIEW

File: migrations/Version20180711070226.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('CREATE TABLE sylius_invoicing_plugin_tax_item (id INT AUTO_INCREMENT NOT NULL, invoice_id VARCHAR(255) NOT NULL, `label` VARCHAR(255) NOT NULL, amount INT NOT NULL, INDEX IDX_2951C61C2989F1FD (invoice_id), PRIMARY KEY(id)) DEFAULT CHARACTER SET UTF8 COLLATE UTF8_unicode_ci ENGINE = InnoDB');
+        $this->addSql('CREATE TABLE sylius_invoicing_plugin_shop_billing_data (id INT AUTO_INCREMENT NOT NULL, company VARCHAR(255) NOT NULL, tax_id VARCHAR(255) NOT NULL, street VARCHAR(255) NOT NULL, city VARCHAR(255) NOT NULL, postcode VARCHAR(255) NOT NULL, country_code VARCHAR(255) NOT NULL, PRIMARY KEY(id)) DEFAULT CHARACTER SET UTF8 COLLATE UTF8_unicode_ci ENGINE = InnoDB');
+        $this->addSql('ALTER TABLE sylius_invoicing_plugin_tax_item ADD CONSTRAINT FK_2951C61C2989F1FD FOREIGN KEY (invoice_id) REFERENCES sylius_invoicing_plugin_invoice (id) ON DELETE CASCADE');
+        $this->addSql('DROP TABLE migration_versions');
```
Comment (bartoszpietrzak1994) [inline migrations/Version20180711070226.php]: Do we need this statement?

File: migrations/Version20180711070226.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        $this->addSql('CREATE UNIQUE INDEX UNIQ_16C8119E5CDB2AEB ON sylius_channel (billing_data_id)');
+    }
+
+    public function down(Schema $schema) : void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE sylius_channel DROP FOREIGN KEY FK_16C8119E5CDB2AEB');
+        $this->addSql('CREATE TABLE migration_versions (version VARCHAR(255) NOT NULL COLLATE utf8_unicode_ci, PRIMARY KEY(version)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB');
```
Comment (bartoszpietrzak1994) [inline migrations/Version20180711070226.php]: Do we need this statement?

File: spec/Entity/TaxItemSpec.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+namespace spec\Sylius\InvoicingPlugin\Entity;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Resource\Model\ResourceInterface;
+use Sylius\InvoicingPlugin\Entity\InvoiceInterface;
+use Sylius\InvoicingPlugin\Entity\TaxItemInterface;
+
+final class TaxItemSpec extends ObjectBehavior
+{
+    function let(): void {
```
Comment (bartoszpietrzak1994) [inline spec/Entity/TaxItemSpec.php]: Move bracket to the next line

File: migrations/Version20180711070226.php (hunk 1 above)
Comment (GSadee) [reply migrations/Version20180711070226.php]: To be honest, I have no idea, it was generated automatically /cc @Zales0123 wdyt?

**Pull request merged**